setup.py

```diff
@@ -50,6 +50,14 @@ def read(*parts):
     with codecs.open(filename, encoding='utf-8') as fp:
         return fp.read()
 
+def get_dependencies(file):
+    root = os.path.dirname(os.path.realpath(__file__))
+    requirements = os.path.join(root, file)
+    result = []
+    if os.path.isfile(requirements):
+        with open(requirements) as f:
+            result = f.read().splitlines()
+    return result
```

Comment:
It could be easier to understand that something is going wrong:
```suggestion
def get_dependencies(file):
    root = os.path.dirname(os.path.realpath(__file__))
    requirements = os.path.join(root, file)
    if os.path.isfile(requirements):
        with open(requirements) as f:
            return f.read().splitlines()
    raise RuntimeError("Unable to get dependencies from file " + file)
```

Reply:
Thanks, added